packages/idyll-cli/package.json

```diff
@@ -40,6 +40,7 @@
     "bs-pretty-message": "^1.0.8",
     "chalk": "^2.4.1",
     "change-case": "^3.0.1",
+    "command-line-args": "^5.1.1",
```

Comment:
Is this used anywhere?

Reply:
Yes used in index.js to read the env param from command line. Is it OK to use 5.1.1 or should I change it to the one already in yarn.lock 5.0.0? If using 5.1.1 I will re-run yarn.lock like you say to do

Reply:
Ah okay 👍 keeping `^5.1.1` is fine in that case